src/guide/component-basics.md

```diff
@@ -1,5 +1,7 @@
 # Components Basics
 
+<VideoLesson href="https://vueschool.io/courses/vue-js-3-components-fundamentals?friend=vuejs" title="Free Vue.js Components Fundamentals Course">Learn component basics with a free video course on Vue School</VideoLesson>
```

Comment:
In **vue-3-intro-to-components**, around 2:05, it says:

> Notice here, that the data is not an object, as it can be in the root Vue instance... etc.

In Vue 3 this is no longer true. `data` has to be a function in all cases. There isn't a special case for the root component.